Podfile.lock

```diff
@@ -83,6 +83,6 @@ SPEC CHECKSUMS:
   WordPress-Ratings-iOS: 9f83dbba6e728c5121b1fd21b5683cf2fd120646
   ZIPFoundation: b1f0de4eed33e74a676f76e12559ab6b75990197
 
-PODFILE CHECKSUM: 3211f27dc676386a9b5d29a814f6e9da30c3e316
+PODFILE CHECKSUM: 697a54e744889aad136325940eda65b46ea401b7
```

Comment:
This checksum changed because of the trailing whitespace removal above, which I committed after checking the `Podfile` to make sure it didn't request any pod by `ref`.